TLM/TLM/UI/Helpers/SerializableUIOptionBase.cs

```diff
@@ -53,18 +52,21 @@ public abstract class SerializableUIOptionBase<TVal, TUI, TComponent> : ILegacyS
 
             if (scope.IsFlagSet(Options.PersistTo.Savegame)) {
                 _fieldInfo = typeof(Options).GetField(fieldName, BindingFlags.Static | BindingFlags.Public);
-
-                if (_fieldInfo == null) {
-                    throw new Exception($"SerializableUIOptionBase.ctor: `{fieldName}` does not exist");
-                }
+                getValue_ = _fieldInfo != null ? GetFieldValue : GetBuiltInValue;
```

Comment:
What is the reason of removed code throwing the `Exception` if `null`? All savegame options should have a field in the `Options` class for fast access so I don't see a reason for trying to speed up this code if that was your intention.
_The code outside UI should not try to read the value of setting from the UI components because... settings shouldn't be stored in UI components_ -> #1372 .

Reply:
Abut #1372 the conversation in the comments have been inconclusive. But we were considering that maybe we should store values in here. maybe the solution could be to move the UI fields outside of the UI class. I should not have dropped the `[CanBeNull]` tag from the field info though.

anyways we currently use this when reading global options so we might as well speed it up.

the change in this file is out of scope of this PR so I will revert it.